src/universal/containers/EditingStatus/EditingStatusContainer.js

```diff
@@ -103,16 +114,28 @@ export default class EditingStatusContainer extends Component {
     clearTimeout(this.refreshTimer);
   }
 
+  toggleTimestamp() {
+    const {timestampType} = this.state;
+    const {editors, active, updatedAt, createdAt} = this.props;
+    const newTimestampType = timestampType === 'updatedAt' ? 'createdAt' : 'updatedAt';
+    const timestamp = newTimestampType === 'updatedAt' ? updatedAt : createdAt;
+    this.setState({
+      timestampType: newTimestampType,
+      editingStatus: makeEditingStatus(editors, active, timestamp, newTimestampType)
+    });
+  }
+
   render() {
-    const {active, editors, updatedAt} = this.props;
-    const {editingStatus} = this.state;
+    const {active, editors, updatedAt, createdAt} = this.props;
+    const {editingStatus, timestampType} = this.state;
+    const timestamp = timestampType === 'updatedAt' ? updatedAt : createdAt;
     clearTimeout(this.refreshTimer);
-    const refreshPeriod = getRefreshPeriod(updatedAt);
+    const refreshPeriod = getRefreshPeriod(timestamp);
     this.refreshTimer = setTimeout(() => {
       this.setState({
-        editingStatus: makeEditingStatus(editors, active, updatedAt)
+        editingStatus: makeEditingStatus(editors, active, timestamp, timestampType)
       });
     }, refreshPeriod);
-    return <EditingStatus status={editingStatus} />;
+    return <EditingStatus status={editingStatus} handleClick={this.toggleTimestamp} />;
```

Comment:
To bind `this`, change this here to `handleClick={() => this.toggleTimestamp()}` and the `this` will be bound for you by the arrow function.